Pcap++/header/PcapFilter.h

```diff
@@ -80,6 +81,7 @@ namespace pcpp
 	protected:
 		bpf_program* m_program;
 		std::string m_lastProgramString;

```

Comment:
Can you also change these variable names to begin with `m_<capital letter>`?

Reply:
Done.